pages/api/account/statistics.js

```diff
@@ -24,9 +24,19 @@ export default async function handler(req, res) {
   res.status(200).json(data);
 }
 
-export async function getStats(username) {
+export async function getStats(username, numberOfDays) {
```

Comment:
I think having a default will simply the code below

```suggestion
export async function getStats(username, numberOfDays = 30) {
```